docs/documentation/api-specification/ion-api.md

```diff
@@ -42,7 +41,49 @@ public class SampleExtension extends ExtendM3Trigger {
     this.ion = ion
     this.logger = logger
   }
+```
+JSON slurper parses text or reader content into a data structure of lists and maps.
+
+```groovy
+    JsonSlurper slurper = new JsonSlurper()
```

Comment:
So the idea is the show the JSON first and then how to parse it, in this case, the JSON looks something like this

```json
{
  "SupportedFileFormats": [
    "pdf",
    "jpg"
  ]
}
```